modules/ezcontent_block/modules/ezcontent_block_cards/ezcontent_block_cards.module

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @file
+ * Attaching js.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_alter().
+ */
+function ezcontent_block_cards_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if ($form_id === 'layout_builder_add_block') {
+    $blockComponent = $form_state->get('layout_builder__component');
+    $pluginId = $blockComponent->getPluginId();
```

Comment:
should we avoid the assignment of variable here?
`if ($blockComponent->getPluginId() === 'inline_block:cards') {`

Rest, LGTM